src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelectorSelectedOption.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Frame} from 'app/types/events';
+import TextOverflow from 'app/components/textOverflow';
+/* eslint-disable react/prop-types */
```

Comment:
Can we move this to the top of the file?

Reply:
oh i didn't notice it before..good catch!